go/hyperkit.go

```diff
@@ -367,32 +375,49 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	a = append(a, "-m", fmt.Sprintf("%dM", h.Memory))
 
 	a = append(a, "-s", "0:0,hostbridge")
+	a = append(a, "-s", "31,lpc")
+
+	nextSlot := 1
```

Comment:
I'm just concerned that we add more and more features to the API, but if it will be used for Docker for Mac porting that's fine with me. Should just remember to remove it after because it's like going to bitrod.